lisa/features/gpu.py

```diff
@@ -137,6 +137,9 @@ def install_compute_sdk(self, version: str = "") -> None:
                     except Exception as e:
                         raise LisaException(f"Failed to install CUDA Driver {str(e)}")
                     self.gpu_vendor.add("nvidia")
+            elif driver == ComputeSDK.AMD:
+                # refer https://learn.microsoft.com/en-us/azure/virtual-machines/nvv4-series  # noqa: E501
+                raise SkippedException("AMD GPU is not supported in Linux yet.")
```

Comment:
I cannot find any information about AMD GPU doesn't support Linux in this page. If so, please clarify, it's not supported on Azure, or general not support Linux.

Reply:
should be this link https://learn.microsoft.com/en-us/azure/virtual-machines/sizes-gpu

![image](https://github.com/microsoft/lisa/assets/10083705/446c5e58-7d68-44dc-8b60-b99ca4311ff9)

Reply:
this page https://learn.microsoft.com/en-us/azure/virtual-machines/nvv4-series also refers to Windows only
![image](https://github.com/microsoft/lisa/assets/10083705/53cb7d32-cf85-4fb0-9d47-2ba7de5be449)

Reply:
Good to know. I think it means the NVv4 series supports windows only. it doesn't mean the AMD GPU supports Windows only. Can we exclude the VM sizes in GPU feature, and add comment to explain there?